manuscript/dockerhub.md

```diff
@@ -0,0 +1,105 @@
+# Dockerhub
+
+You are learning to do pretty cool stuff, building your image, what about sharing with others what you are doing? This can be helpful for you in the future, this is how communities grow strong. That's why **Dockerhub** exists.
+
+#### What is Dockerhub?
+
+Dockerhub is a shared repository of Docker images. In the previous chapter, you read a little about official images and not-official, is on Dockerhub that the **Dockerfiles** are distributed for the community with the object to facilitate the build of your docker environment.
+
+Several companies make their official product available as docker images, as:
```

Comment:
based on this article: https://docs.docker.com/docker-hub/official_images/  the official images are curated by docker. The Companies don't makes their image :) It is done my Docker team.